### ISSUE TITLE
feat: logToConsole

### ISSUE THREAD

**Issue by dqn** [mayteio/use-pusher#37]
```js
import React from "react";
import { PusherProvider } from "@harelpls/use-pusher";
 
const config = {
  // ...
  logToConsole: true,
  // ...
};
 
const App = () => {
  <PusherProvider {...config}>
    <Example />
  </PusherProvider>;
};
```

I want this to be:

```js
Pusher.logToConsole = true;
```

**Comment by mayteio** [comment]
Hey @dqn the provider already passes on all arguments to the `new Pusher({..., ...props})` :

https://github.com/mayteio/use-pusher/blob/8622e1d0795c1c8bcbeb4c8e49ff4f8c6264a8d7/src/core/types.d.ts#L11

if it’s something that isn’t an argument, I’d suggest using the usePusher hook:

```
const { client } = usePusher()
client?.logToConsole = true
```

**Comment by dqn** [comment]
Solved by [#38 (comment)](https://github.com/mayteio/use-pusher/pull/38#issuecomment-767229749)

**Comment by melkir** [comment]
This doesn't seems to work properly 🤔
